### DISCUSSION TITLE
Thankyou

### DISCUSSION THREAD

**Discussion by Bobspadger** [jwillinghalpern/filemaker-vscode-bundle#10] [General]
Just thought I would drop a quick note of thanks for the effort you have put in on this.

Just for the JSON to JSONSetElement helper so I no longer need to tear my hair out making horrendous Json create scripts in the terrible Filemaker script editor !

Thanks so much!

**Comment by jwillinghalpern** [comment]
Hey thanks for the kind words, I'm happy to do it, and I like the opportunity to learn more about how vscode works.

I've had that json to jsonsetelement function for a while as a Mac automator service, but I wasn't sure how best to share it with others since automator is clunky and mac only. I hope it saves some frustration!

J
